webapp/packages/plugin-authentication/src/Dialog/useAuthDialogState.ts

```diff
@@ -173,6 +173,9 @@ export function useAuthDialogState(accessRequest: boolean, providerId: string |
         provider = (provider || state.activeProvider) ?? undefined;
         configuration = (configuration || state.activeConfiguration) ?? undefined;
 
+        state.credentials.credentials.user = state.credentials.credentials.user?.trim();
+        state.credentials.credentials.password = state.credentials.credentials.password?.trim();
```

Comment:
same here, please don't make modifications in functions like submit or save

form data -> prepare data -> submit data to the server -> load form data from the server

why?: let's imagine that we modified data before submitting to the server and now it's invalid or creates an exception on server side (user can't modify form to fix this error)